test/utils/delay.ts

```diff
@@ -0,0 +1,3 @@
+export default function delay(time: number) {
+	return new Promise(resolve => setTimeout(resolve, time));
```

Comment:
pretty sure jest already has this built in.

Reply:
For reasons explained above, we have to wait